app/src/main/java/com/github/se/icebreakrr/ui/profile/OtherProfileView.kt

```diff
@@ -259,11 +270,35 @@ fun BottomSheet(aiState: AiViewModel.UiState, onDismissRequest: () -> Unit) {
               when (aiState) {
                 is AiViewModel.UiState.Success -> {
                   Text(
-                      text = "\"${aiState.data}\"",
+                      text = "«${aiState.data}»",
                       fontWeight = FontWeight.Normal,
                       fontSize = TextUnit(CONTENT_FONT_SIZE, TextUnitType.Sp),
                       lineHeight = TextUnit(CONTENT_LINE_HEIGHT, TextUnitType.Sp),
                       modifier = Modifier.testTag("aiResponse"))
+
+                  Spacer(modifier = Modifier.height(30.dp))
```

Comment:
That's a magic number there my friend!

Reply:
After this project, these two words will definitely trigger a PTSD